Sources/UI/SwiftUI/Views/SimpleColorPicker/SimpleColorPicker.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SimpleColorPicker.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 15/08/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+#if os(macOS)
+import SwiftUI
+import Combine
+
+/// SwiftUI wrapper for NSColorWell, allowing the selection of colors with a customizable size.
+struct SimpleColorPicker: NSViewRepresentable {
```

Comment:
You probably want it public.

Reply:
[Indeed](https://github.com/poviolabs/PovioKit/pull/332/commits/6c0450ac9ddb0bdf1baa7174ae03d73a46abc3d4).